advanced_animation/core/storyboard_generator.py

```
@@ -270,6 +245,185 @@ def _generate_ai_storyboard(self, code_analysis: Dict[str, Any]) -> Storyboard:
+                    all_scenes.extend(chunk_response.get('scenes', []))
+                else:
+                    logger.warning(f"Failed to get response for chunk {i//chunk_size + 1}")
+            
+            # Combine all scenes into final storyboard
+            final_storyboard = {
+                'title': 'Comprehensive Code Repository Analysis',
+                'description': 'Detailed educational animation with code execution flow, AST analysis, and algorithm visualization',
+                'scenes': all_scenes
+            }
+            
+            return final_storyboard
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)